src/components/userAuth/signInMail/forgotPassword/resetPassword/PasswordHintColorAndIcon.tsx

```diff
@@ -0,0 +1,95 @@
+import { Box, FormHelperText, List, ListItem, Typography } from '@mui/material'
+import { makeStyles } from 'tss-react/mui'
+
+const PasswordHintColorAndIcon = ({ password }: { password: string }) => {
+  const { classes } = useStyles()
+
+  const passwordHintColor = (pattern: RegExp, password: string) => {
+    if (password) {
```

Comment:
looks a bit like spaghetti code, can't offer another solution right now, but maybe you will be able to do changes. Not critical but though

Reply:
Unfortunately, I don't have a better idea now. Maybe something come later